guides/source/active_record_querying.md

```diff
@@ -1441,7 +1440,7 @@ Produces:
 ```sql
 SELECT customers.* FROM customers
 LEFT OUTER JOIN reviews ON reviews.customer_id = customers.id
-WHERE reviews.customer_id IS NULL
```

Comment:
It was wrongly documented by https://github.com/rails/rails/commit/d7cfaeb1528abefdc6f27c21feb748ad715182c9